sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsActions.java

```diff
@@ -174,6 +175,31 @@ public TextAnalyticsActions setAnalyzeSentimentActions(AnalyzeSentimentAction...
         return this;
     }
 
+    /**
+     * Gets the list of {@link ExtractSummaryAction} to be executed.
+     *
+     * @return the list of {@link ExtractSummaryAction} to be executed.
+     */
+    public Iterable<ExtractSummaryAction> getExtractSummaryActions() {
+        return extractSummaryActions;
+    }
+
+    /**
+     * Sets the list of {@link ExtractSummaryAction} to be executed.
+     *
+     * @param extractSummaryActions The list of {@link ExtractSummaryAction} to be executed.
+     *
+     * @return The {@link TextAnalyticsActions} object itself.
+     *
+     * @throws IllegalArgumentException if more than one {@link ExtractSummaryAction} action are passed in.
+     * Currently service v3.2-preview.1 only accepts up to one action per type.
```

Comment:
should we change all the version numbers for the rest of the actions where this text is present?

Reply:
if we set it to v3.2-preview.1 what about user is using v3.1?

Reply:
hmm good point. maybe saying v3.1 and up? 
otherwise a user could think that if they use 3.2-preview.1 that they can set more than one action